src/pu-main.c

```diff
@@ -1,23 +1,28 @@
 /*
  * SPDX-License-Identifier: GPL-3.0-or-later
- * Copyright (c) 2022 PHYTEC Messtechnik GmbH
+ * Copyright (c) 2023 PHYTEC Messtechnik GmbH
  */
 
+#define G_LOG_DOMAIN "partup"
+
 #include <glib.h>
 #include <glib/gstdio.h>
 #include <parted/parted.h>
 #include <unistd.h>
 #include "pu-config.h"
 #include "pu-emmc.h"
 #include "pu-flash.h"
+#include "pu-log.h"
 #include "pu-mount.h"
 #include "pu-utils.h"
 #include "pu-version.h"
 
 static gboolean arg_debug = FALSE;
+static gboolean arg_quiet = FALSE;
 static gboolean arg_version = FALSE;
 static gboolean arg_skip_checksums = FALSE;
 static gchar *arg_config = NULL;
+static gchar *arg_debug_domains = FALSE;
```

Comment:
`arg_debug_domains` is not a boolean value, but a string. Initialize it with `NULL`.